src/utils/fetchTransactions.ts

```diff
@@ -9,8 +10,8 @@ const providerURL = process.env.PROVIDER_URL || 'https://rpc.gaiaxtestnet.oceanp
 const provider = new ethers.providers.JsonRpcProvider(providerURL)
 
 async function getLatestTxBlockNumber() {
-  const txArray = await Transaction.findOne({}).sort('-blockNumber').exec()
-  return txArray?.blockNumber
+  const txArray = await Transaction.find().sort({ blockNumber: -1 }).limit(1)
+  return txArray?.[0]?.blockNumber
```

Comment:
question(blocking): What is the reason behind changing from `findOne` to `find`?

Reply:
I checked the queries for problems and found that this query is more performant.